pkg/security/security_profile/profile/profile.go

```diff
@@ -203,6 +204,9 @@ func LoadProtoFromFile(filepath string) (*proto.SecurityProfile, error) {
 	if err = pp.UnmarshalVT(raw); err != nil {
 		return nil, fmt.Errorf("couldn't decode protobuf profile: %w", err)
 	}
+
+	seclog.Errorf("profile loaded from %s (%d bytes) => %s:%s", filepath, len(raw), pp.Selector.ImageName, pp.Selector.ImageTag)
```

Comment:
should we remove this debug log ? (or, put it as Info instead?)

Reply:
ah yes, I'm going to keep it for now because it's useful to understand the amount of dumps loaded